package execptionhandling;

//always System.out.println("aa"); ll be printed then only we will think abt try - catch - what o/p ll come and others



public class Dummythrows {	
	
	
	
	
	

	//public static void main(String[] args){
		/*
	  System.out.println("aa");
	
	
		int a=10;
		int c=a/0;
		
		System.out.println(c);
		*/
		
		
		
		
	/*
	 * aa
Exception in thread "main" java.lang.ArithmeticException: / by zero
	at execptionhandling.Dummythrows.main(Dummythrows.java:13)

	 * 
	 * 
	 * 
	 * 	
	 */
		
		//*******************************************************************************
		
		/*
		  System.out.println("aa");
			
			
			int a=10;
			int c=a/0;
			
			System.out.println(c);
		
			System.out.println("aa1");
		*/
		
			 
			 
			 
			
			/*
			 * aa
Exception in thread "main" java.lang.ArithmeticException: / by zero
	at execptionhandling.Dummythrows.main(Dummythrows.java:37)

			 * 
			 * 
			 * 
			 * 
			 */
			  
		
		
		//*******************************************************************************
		
		
		/*
		public static void main(String[] args) throws ArithmeticException{
			
		System.out.println("aa");
		
		
		int a=10;
		int c=a/0;
		
		System.out.println(c);
	
		System.out.println("aa1");
		*/
	
	
	
	
		
		/*
		 * 
		 * aa
Exception in thread "main" java.lang.ArithmeticException: / by zero
	at execptionhandling.Dummythrows.main(Dummythrows.java:73)

		 * 
		 * 
		 * 
		 * 
		 * 
		 */
		
	
	
		
	
		
		
	}

}
